strings/ion_laws.json

```diff
@@ -535,10 +521,10 @@
         "PRESS B",
         "SMELL LIKE THE MAN YOUR MAN COULD SMELL LIKE",
         "PIRATE VIDEO GAMES",
-        "WATCH PORNOGRAPHY",
         "INSULT THE LIZARDS",
         "FLIRT WITH THE LIZARDS",
-        "GAS THE LIZARDS"
+		"INSULT THE ETHARI",
+		"FLIRT WITH THE ETHARI"
```

Comment:
:eyes:

Reply:
I wasn't going to add it, but it existed for lizards. Yeah those could both go, I'd be more than happy with that.

Reply:
I'm not saying remove it, I'm just surprised you put it there

Reply:
*I am too*

Reply:
🍆